CHANGELOG.md

```diff
@@ -310,18 +442,102 @@
   [#3113](https://github.com/mixxxdj/mixxx/pull/3113)
   [#3089](https://github.com/mixxxdj/mixxx/pull/3089)
   [#3545](https://github.com/mixxxdj/mixxx/pull/3545)
-* Add support for saving loops as hotcues
-  [#2194](https://github.com/mixxxdj/mixxx/pull/2194)
-  [#4265](https://github.com/mixxxdj/mixxx/pull/4265)
-  [lp:1367159](https://bugs.launchpad.net/mixxx/+bug/1367159)
-* Make use of inclusive language
+* Update VCPK build environment for Windows and MacOS
+  [#4163](https://github.com/mixxxdj/mixxx/pull/4163)
+  [#4225](https://github.com/mixxxdj/mixxx/pull/4225)
+  [#4338](https://github.com/mixxxdj/mixxx/pull/4338)
+* Devendor libraries from the mixxx lib folder
+  [#4201](https://github.com/mixxxdj/mixxx/pull/4201)
+  [#4202](https://github.com/mixxxdj/mixxx/pull/4202)
+* Update Benchmark library to v1.6.0 [#4540](https://github.com/mixxxdj/mixxx/pull/4540)
+* Migration to Qt6 (work in progress)
```

Comment:
Do we really need to mention every single PR?

Reply:
Looking at the final rendering, it becomes a single line like this: 

Migration to Qt6 (work in progress) #4052 #4295 #4293 #4294 #4291 #4290 #4300 #4302 #4289 #4292 #4299 #4051 #4303 #4305 #4304 #4306 #4308 #4309 #4322 #4373 #4371 #4375 #4378 #4381 #4380 #4376 #4379 #4372 #4377 #4387 #4391 #4392 #4395 #4397 #4396 #4402 #4405 #4394 #4404 #4401 #4400 #4403 #4407 #4399 #4406 #4420 #4415 #4417 #4419 #4416 #4418 #4433 #4434 #4441 #4445 #4446 #4444 #4436 #4437 #4440 #4430 #4435 #4443 #4439 #4442 #4438 #4449 #4451 #4453 #4478 #4479 #4506 #4556 #4554 #4555 #4552 #4549

So this is kind of OK for now as a work in progress solution. It would work for me to list the single PRs somewhere else or drop the whole list at all.

Reply:
thats still a really long line, I'd also prefer to just link the project.

Reply:
The project is a moving target though. Since the whole topic does not really has relevance for the user in a 2.4 release, we can also consider to ditch the whole line. 
I propose to postpone the decision to the final polish of this file before the actual release. 
OK?

Reply:
I think we want to keep the changelog focused on user-visible changes, and because this work isn't visible to users yet, I would argue for not including the line at all.  The eventual changelog line we want is "Added support for QT6" and then we'd link to the project. Since it's such a major undertaking, it doesn't seem helpful to include every PR.  (These are just my thoughts, I don't have a specific requirement for this part)